### ISSUE TITLE
🚀 Feature: Setting default language for all non logged in and new users + language icon

### ISSUE THREAD

**Issue by returnedinformation** [stonith404/pingvin-share#687]
### 🔖 Feature description

Having an option within settings to choose which language of the UI will be used by default for all new and non-logged in users.
Having a language icon at the main page to quickly change UI language.

### 🎤 Pitch

I'm really glad you included Serbian translations :D
But now I have an issue. I wish to set Serbian (Latin) as a language for all non logged users, because in my case, most of people that'll be using it don't know English well.

Also, as a suggestion, it would probably be a good idea to put some kind of a 🌎icon which will open up a dropdown menu of which language to use, so people can see it immediately and change if they want to.

**Comment by returnedinformation** [comment]
I've just realized it takes the language set inside the Web Browser. That's great, but in my case some computers have it set to Serbian, while others have it set to Croatian... I've submitted a request to do a Croatian translation as well, it's similar to Serbian Latin

**Comment by stonith404** [comment]
Yes, it uses the browser’s language setting and defaults to English if the selected language is unavailable. For this reason, I wouldn’t consider this issue a high priority, but I will keep it open for now.

**Comment by GitNees** [comment]
yeah would be great, on my PC is in english, what I want, on my phone its in other language when browse to my pingvin, its weird, i want it in english, but cant be set now it seems
